src/drive/share/index.ts

```diff
@@ -95,6 +96,16 @@ export class Share {
     return this.client.delete(`/storage/share/${shareId}`, this.headers());
   }
 
+  /**
+   * Delete share links by id
+   * @param payload
+   */
+  public deleteShareLinks(
+    payload: ShareItemsIds,
+  ): Promise<{ deleted: boolean; items: Array<{ id: number | string }> }> {
+    return this.client.delete('/storage/share/', { items: JSON.stringify(payload.items) }, this.headers());
```

Comment:
Is there any reason or advantage to use stringify instead of passing the elements as a normal array?

Reply:
It was a mistake, I've fixed